src/backend/core/api/viewsets/service.py

```diff
@@ -164,23 +166,47 @@ class OrganizationServiceViewSet(viewsets.ReadOnlyModelViewSet):
     queryset = models.Service.objects.all()
     serializer_class = serializers.OrganizationServiceSerializer
     permission_classes = [
-        IsAuthenticated,
+        permissions.IsAuthenticatedWithAnyMethod,
         permissions.ParentOrganizationAccessPermission,
```

Comment:
I think we should verify that the `organization_id` and `operator_id` matches, because if a user is logged-in we only check for role on organization.

But we could provide another `operator_id` in the url 🤔 Not directly related to this PR though.